congress_lookup.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+#coding: utf-8
+__author__ = 'stsmith'
+
+# congress_lookup: Look up information about congress from the congress-legislators database
+# See: https://github.com/unitedstates/congress-legislators and https://github.com/TheWalkers/congress-legislators
+
+# The project is in the public domain within the United States, and
+# copyright and related rights in the work worldwide are waived
+# through the CC0 1.0 Universal public domain dedication.
+
+# Author 2017 Steven T. Smith <steve dot t dot smith at gmail dot com>
+
+import argparse as ap, contextlib, errno, fnmatch, os, urllib2, urlparse, yaml
+
+class CongressLookup:
+    '''A class used to lookup legislator properties from the github congress-legislators YAML database.'''
+
+    def __init__(self):
+        self.args = self.parseArgs()
+        self.data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),self.args.data_dir)
+        self.properties = dict()
+        self.database_load()
+        for prop in self.args.properties: self.lookup_property(prop)
+
+    def parseArgs(self):
+        parser = ap.ArgumentParser()
+        parser.add_argument('properties', metavar='PROPS', type=str, nargs='+',
+                            help='Properties to look up')
+        parser.add_argument('-c', '--committee', help="Committee name (wildcard)", type=str, default=None)
+        parser.add_argument('-n', '--last-name', help="Last name of legislator (wildcard)", type=str, default=None)
+        parser.add_argument('-d', '--data-dir', help="Database directory", type=str, default='.')
+        parser.add_argument('-r', '--repo', help="GitHub repo URL", type=str, default='https://github.com/essandess/congress-legislators/')
```

Comment:
If it's coming into the repo, the default GitHub repo URL should be `unitedstates/congress-legislators`.